generators/python/src/fern_python/generators/fastapi/service_generator/endpoint_generator.py

```diff
@@ -183,15 +183,6 @@ def _write_init_body(writer: AST.NodeWriter) -> None:
             )
             writer.write_line()
 
-            writer.write_line("# this is necessary for FastAPI to find forward-ref'ed type hints.")
```

Comment:
Note that this was only fixed recently in 0.123.5 https://github.com/fastapi/fastapi/pull/5077

Although our existing code wasn't perfect and had some bugs. What do you think we should do to make sure that the break here is minimal?